x-pack/auditbeat/module/system/user/users_linux.go

```diff
@@ -6,18 +6,26 @@
 
 package user
 
+// #include <errno.h>
 // #include <sys/types.h>
 // #include <pwd.h>
 // #include <shadow.h>
+//
+// void setErrno(int err) {
```

Comment:
How about `clearErrno()` with no args? Is there any reason to set to a non-zero value?

Reply:
Not in this case. I've changed it.